polimec-common/test-utils/src/lib.rs

```diff
@@ -60,26 +67,41 @@ pub fn get_mock_jwt<AccountId: frame_support::Serialize>(
 		.as_ref(),
 	)
 	.unwrap();
-	// We don't need any custom fields in the header, so we use the empty.
+
 	let header: Header = Header::empty();
 
-	// Create the custom part of the `Claims` struct.
-	let custom_claims: SampleClaims<AccountId> =
-		SampleClaims { subject: account_id, investor_type, issuer: "verifier".to_string(), did };
+	// Handle optional IPFS CID
+	let ipfs_cid = ipfs_cid.unwrap_or_else(|| BoundedVec::with_bounded_capacity(96));
+
+	let custom_claims =
+		SampleClaims { subject: account_id, investor_type, issuer: "verifier".to_string(), did, ipfs_cid };
 
-	// Wrap the SampleClaims` struct in the `Claims` struct.
 	let mut claims = Claims::new(custom_claims);
-	// Set the expiration date to 2030-01-01.
-	// We need to unwrap the `Utc::with_ymd_and_hms` because it returns a `LocalResult<DateTime<Utc>>` but we ned a `DateTime<Utc>.
 	claims.expiration = Some(Utc.with_ymd_and_hms(2030, 1, 1, 0, 0, 0).unwrap());
 
-	// Create a JWT using the Ed25519 algorithm.
 	let token_string = Ed25519.token(&header, &claims, &key).unwrap();
-
-	// Create an `UntrustedToken` from the signed JWT string.
 	UntrustedToken::new(&token_string).expect("Failed to parse the JWT")
 }
 
+// The `Serialize` trait is needed to serialize the `account_id` into a  `SampleClaims` struct.
+pub fn get_mock_jwt<AccountId: frame_support::Serialize>(
+	account_id: AccountId,
+	investor_type: InvestorType,
+	did: BoundedVec<u8, ConstU32<57>>,
+) -> UntrustedToken {
+	create_jwt(account_id, investor_type, did, None)
+}
+
+// The `Serialize` trait is needed to serialize the `account_id` into a  `SampleClaims` struct.
+pub fn get_mock_jwt_with_cid<AccountId: frame_support::Serialize>(
+	account_id: AccountId,
+	investor_type: InvestorType,
+	did: BoundedVec<u8, ConstU32<57>>,
+	ipfs_cid: BoundedVec<u8, ConstU32<96>>,
+) -> UntrustedToken {
+	create_jwt(account_id, investor_type, did, Some(ipfs_cid))
+}
+
```

Comment:
Why not have just one function where ipfs_cid is an `Option<Cid>`?

Reply:
That was my initial approach, but it caused all the tests to fail. To avoid modifying all the tests at this early stage of implementation, I opted for a different function